src/AtlasModules/README.md

```diff
@@ -2,17 +2,17 @@
 To verify the legitmacy of these binaries (executables), there's information provided here for each binary for hashes, sources and when each was last verified/checked with the SHA256 hash. Verified using `Get-FileHash` in PowerShell. 
 
 ### DevManView
-- SHA256 Hash: `A1BE60039F125080560EDF1EEBEE5B6D9E2D6039F5F5AC478E6273E05EDADB4B`
+- SHA256 Hash: `6FCDEC03A8E91D77D4682E9839622D8C42F5B47BD0DCD19DDC7F5F4764EDB7DA`
```

Comment:
This should be the hash of the executable (`devmanview.exe`), not the `.zip` that it is downloaded in.

Hash of the executable: `fc81134cfb57cd64a76beeac74cfab29768d3b542ec6b25429f5337780790a1e`

The updated executable in this pull request does **not** match this though, and it has a file size difference of 50kb.

![explorer_W0CNaL6O4O](https://user-images.githubusercontent.com/65787561/205135763-39dd25cc-dd43-4bfc-b62b-bee6b4bc3a26.png)

Reply:
Thought I had the correct one, thanks for the recorrection, will fix now!

Reply:
Fixed